core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import io.vertx.core.json.DecodeException;
+import org.apache.qpid.proton.message.Message;
+
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Constants &amp; utility methods used throughout the Tenant API.
+ */
+
+public final class TenantConstants extends RequestResponseApiConstants {
+
+    /* tenant actions */
+    public static final String ACTION_GET                        = "get";
+    public static final String ACTION_ADD                        = "add";
+    public static final String ACTION_UPDATE                     = "update";
+    public static final String ACTION_REMOVE                     = "remove";
+
+
+    /* message payload fields */
+    public static final String FIELD_ENABLED                     = "enabled";
```

Comment:
`RequestResponseApiConstants` also defines a constant of this name. IMHO we should remove it here since we use the *enabled* field consistently across all APIs.

Reply:
:+1: